cachi2/core/package_managers/generic/models.py

```diff
@@ -1,18 +1,25 @@
 import re
+import urllib
```

Comment:
urllib import is couple lines below

Reply:
Consolidated into a single urllib import, good catch.